### PR TITLE
Closes issue #194, issue #196, issue #202, issue #206, and issue #207…

### PR DESCRIPTION
**Pull request by dillerm**
…: Fixes definition of 'role in human social processes'; deprecates 'residential function'; creates class 'birth attendant role'; changes label of 'insurance company' to 'insurance organization'; and adds classs 'religious gathering', 'hospital bed capacity data item', 'proportion of population in an elementary occupation', 'Medicaid policy holder role', 'US state Medicaid agency', 'insurance company', and 'private insurance policy holder role'.

### CODE REVIEW

File: src/ontology/Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # ----------------------------------------
 # Makefile for omrse
 # Generated using ontology-development-kit
-# ODK Version: v1.4
+# ODK Version: v1.3.1
```
Comment (matentzn) [inline src/ontology/Makefile:4]: Be careful! You are using an old ODK, run "docker pull obolibrary/odkfull" and then update_repo again

**Comment by hoganwr** [comment]
Done. Plus fixed several minor ROBOT report errors.

**Comment by hoganwr** [comment]
Ok, now I also finished my review. Just committed two minor changes. If
this looks good to you, go ahead and merge.

**Comment by dillerm** [comment]
Looks good to me!

**Pull request merged**